### PR TITLE
Migrate Glimmer tools

### PR DESCRIPTION
**Pull request by abretaud**
Hi,
As discussed in https://github.com/bgruening/galaxytools/pull/660, here's a PR to condafy and update the glimmer tools that are currently in https://github.com/bgruening/galaxytools/

It uses a not-yet-merged new datatype: https://github.com/galaxyproject/galaxy/pull/5042

I had to rename the tools because their id was invalid (`-` in their id), and as I did it, I created a proper tool suite. I kept the owner as bgruening in .shed.yml, but I guess it will be hard to keep the tool lineage... Any thought on that?

Anthony

### CODE REVIEW

File: tools/glimmer/glimmer_acgt_content.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<tool id="glimmer_acgt_content" name="ACGT Content" version="@WRAPPER_VERSION@">
+    <description>of windows in each sequence</description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_acgt_content.xml]: indentation.

File: tools/glimmer/glimmer_acgt_content.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	window-start	window-len	A's	C's	G's	T's	#other	%GC
+
+Note the last window in the sequence can be shorter than *window-len* if the sequence ends prematurely
+
+
+
+
+**References**
+
+A.L. Delcher, K.A. Bratke, E.C. Powers, and S.L. Salzberg. Identifying bacterial genes and endosymbiont DNA with Glimmer. Bioinformatics (Advance online version) (2007).
```
Comment (bgruening) [inline tools/glimmer/glimmer_acgt_content.xml]: can this now be removed if we have a citation?

File: tools/glimmer/glimmer_build_icm.xml [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<tool id="glimmer_build_icm" name="Glimmer ICM builder" version="@WRAPPER_VERSION@">
+    <description></description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_build_icm.xml]: indentation.

File: tools/glimmer/glimmer_extract.xml [hunk 4]
```diff
@@ -0,0 +1,120 @@
+<tool id="glimmer_extract" name="Extract sequence regions" version="@WRAPPER_VERSION@">
+    <description>from a genome</description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_extract.xml]: indentation.

File: tools/glimmer/glimmer_gbk_to_orf.xml [hunk 5]
```diff
@@ -0,0 +1,217 @@
+<tool id="glimmer_gbk_to_orf" name="Extract ORF" version="@WRAPPER_VERSION@">
+    <description>from a GenBank file</description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_gbk_to_orf.xml]: indentation.

File: tools/glimmer/glimmer_glimmer_to_gff.xml [hunk 6]
```diff
@@ -0,0 +1,65 @@
+<tool id="glimmer_glimmer_to_gff" name="Convert Glimmer to GFF" version="@WRAPPER_VERSION@">
+    <description></description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_glimmer_to_gff.xml]: indentation

File: tools/glimmer/glimmer_w_icm.xml [hunk 7]
```diff
@@ -0,0 +1,142 @@
+<tool id="glimmer_knowledge_based" name="Glimmer3" version="@WRAPPER_VERSION@">
+    <description>Predict ORFs in prokaryotic genomes (knowlegde-based)</description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_w_icm.xml]: indentation.

File: tools/glimmer/glimmer_wo_icm.xml [hunk 8]
```diff
@@ -0,0 +1,117 @@
+<tool id="glimmer_not_knowledge_based" name="Glimmer3" version="@WRAPPER_VERSION@">
+    <description>Predict ORFs in prokaryotic genomes (not knowlegde-based)</description>
+    <macros>
+	       <import>macros.xml</import>
```
Comment (bgruening) [inline tools/glimmer/glimmer_wo_icm.xml]: indentation.

**Comment by abretaud** [comment]
Thanks for the review!
Before merging, maybe there is some permissions to grant due to https://github.com/galaxyproject/tools-iuc/pull/1593/files#diff-511019319696751542170ba149cd122eR1

**Comment by bgruening** [comment]
I granted all permission I could to IUC and the genome-annotation group.

**Pull request merged**